vic/drivers/shared_image/src/vic_store.c

```diff
@@ -1666,13 +1676,17 @@ initialize_state_file(char           *filename,
                             &(nc_state_file->node_dimid));
         check_nc_status(status, "Error defining soil_node in %s", filename);
 
+
         if (options.LAKES) {
             status = nc_def_dim(nc_state_file->nc_id, "lake_node",
                                 nc_state_file->lake_node_size,
                                 &(nc_state_file->lake_node_dimid));
             check_nc_status(status, "Error defining lake_node in %s", filename);
         }
 
+        // add extension dimensions
+        initialize_state_file_extension(filename, nc_state_file);
```

Comment:
rename to `inialize_state_file_rout_extension`

Reply:
done